spec/support/features/session_helper.rb

```diff
@@ -31,8 +31,7 @@ def sign_up_with_and_set_password_for(email = nil, reset_session = false)
 
     def sign_up_and_2fa(email = nil, reset_session = false)
       user = sign_up_with_and_set_password_for(email, reset_session)
-      check 'Email'
-      uncheck 'Mobile'
+      full_in 'mobile', with: '555-555-5555'
```

Comment:
spelling error: full_in should be `fill_in`. I'm surprised this didn't cause the test to fail.